android/build.gradle

```diff
@@ -26,3 +26,43 @@ rootProject.ext {
     versionCode = ver.V1
     versionName = ver.V2
 }
+
+tasks.register("lintAllModules") {
+    group = "verification"
+    description = "Run lint on all modules and produce a merged report"
```

Comment:
Would it be easier/cleaner/faster to produce lint reports separately for different modules, only if the related files were changed? Will it require fewer code changes and be easier to read?

Reply:
A full lint report can be helpful if attached to the "good first issue".

Reply:
Separate lint reports are already generated
I have created a Gradle task to to generate a merged report because we need it on CI